src/main/java/com/jtrio/zagzag/product/ProductDto.java

```diff
@@ -6,12 +6,25 @@
 
 @Data
 public class ProductDto {
-    private String name;
-    private Integer price;
-    private String image;
-    private Byte productScore;
-    private Byte deliveryScore;
-    private Long categoryId ;
-    private String categoryName ;
+
+    @Data
+    public static class CreateProductDto {
+        private String name;
+        private Integer price;
+        private String image;
+        private Byte productScore;
+        private Byte deliveryScore;
+        private Category category;
+    }
+
+    @Data
+    public static class OrderProduct{
+        private Long id;
+        private String name;
+        private Integer price;
+        private String image;
+        private Category category;
```

Comment:
dto클래스가 entity클래스를 들고있는건 좋지 않아~ 
사실상 order 와 Product는 엔티티가 분리되어있기 때문에 productcommand의 inneer클래스가 아니라 ordercommand의 innerclass가 되어야해

Reply:
order완료 시에, Product에 대한 정보를 같이 내보내려고 한거라, 
OrderDto에 
```suggestion
        private ProductDto.OrderProduct product;
```
이렇게 추가해주었었거든,

ProductDto.OrderProduct을 ProductDto.OrderProductDto로 바꿔버리면 ?

이름 바꿔도 주문 완료시, 상품정보에 대한 product를 OrderDto에 innerClass로 추가해야됭 ?

Reply:
innerclass가 아니라 그냥 필드로 만들어서 필요한것만 채워줘도돼~